### PR TITLE
Updated LinkedList.cpp

### PR DESCRIPTION
**Pull request by Yash-Thakare22**
Implemented Reversing a  Linked List in C++ and  corrected the error in the program

### CODE REVIEW

**Comment by atarax665** [comment]
I find the changes to be superfluous.

**Comment by Yash-Thakare22** [comment]
Please update my work

**Comment by Yash-Thakare22** [comment]
@iamrajiv Please update my work

File: C++/ds/LinkedList.cpp [hunk 1]
```diff
@@ -115,10 +115,28 @@ int DeleteNode(int pos)   //Takes position to delete the corresponding Node from
     return x;
     }
 }
+
+void ReverseList()  //To Reverse a Linked List
```
Comment (iamrajiv) [inline C++/ds/LinkedList.cpp:119]: @YashThakare I think you have implemented an iterative way of reversing the linked list so mention that in the code comments. Also, add time and space complexity for the same.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@YashThakare update the readme file

**Comment by iamrajiv** [comment]
@YashThakare Please, resolve merge conflicts

**Comment by Yash-Thakare22** [comment]
I will resolve merge conflicts @iamrajiv

**Pull request closed without merging**